freemocap/diagnostics/calibration/calibration_utils.py

```diff
@@ -0,0 +1,91 @@
+from skellytracker.trackers.charuco_tracker.charuco_model_info import CharucoTrackingParams, CharucoModelInfo
+from skellytracker.process_folder_of_videos import process_folder_of_videos
+from pathlib import Path
+from typing import Union
+import numpy as np
+from dataclasses import dataclass
+
+
+@dataclass
```

Comment:
Use Pydantic (v2) BaseModels over dataclasses, whenever possible. Pydantic throws an error on type mismatch, dataclasses don't 

Also - lets align this and any other Stats model with the DescriptiveStatistics model I made recently in `SkellyCam` (eventually)